doc/en/user/source/community/arcsde/index.rst

```diff
@@ -5,7 +5,7 @@ ArcSDE
 
 .. note:: ArcSDE support is not enabled by default and requires the ArcSDE extension to be installed prior to use.  Please see the section on :ref:`arcsde_install` for details.
 
-ESRI's `ArcSDE <http://www.esri.com/software/arcgis/arcsde/>`_ is a spatial engine that runs on top of a relational database such as Oracle or SQL Server.  GeoServer with the ArcSDE extension supports ArcSDE **versions 9.2 and 9.3**.  It has been tested with **Oracle 10g** and **Microsoft SQL Server 2000 Developer Edition**.  The ArcSDE extension is based on the GeoTools ArcSDE driver and uses the ESRI Java API libraries.  See the `GeoTools ArcSDE page <http://docs.geotools.org/latest/userguide/library/data/arcsde.html>`_ for more technical details.
+ESRI's `ArcSDE <http://www.esri.com/software/arcgis/arcsde/>`_ is a spatial engine that runs on top of a relational database such as Oracle or SQL Server.  GeoServer with the ArcSDE extension supports ArcSDE **versions 10.2 and 10.3**.  It has been tested with **Oracle 10g** and **Microsoft SQL Server 2000 Developer Edition**.  The ArcSDE extension is based on the GeoTools ArcSDE driver and uses the ESRI Java API libraries.  See the `GeoTools ArcSDE page <http://docs.geotools.org/latest/userguide/library/data/arcsde.html>`_ for more technical details.
```

Comment:
Will the documentation for arcsde move in geotools? Yup:
https://github.com/geotools/geotools/pull/2771/files#diff-6ec99f02e0b0eec76a4a64682f3181e6L90

Reply:
Actually that link doesn't change. Followed what's done with the other unsupported data plugins, they're listed under the unsupported title in the data page but are all located under library/data:
```

Unsupported plugins:

.. toctree::
   :maxdepth: 1
   
   arcsde
   dxf
   georest
   kml
   solr
   wfs-ng
```

See the links at the bottom of http://docs.geotools.org/latest/userguide/library/data/